workshops/models.py

```diff
@@ -32,6 +33,26 @@ class Meta:
         abstract = True
 
 
+class ActiveMixin(models.Model):
```

Comment:
`.active` was used before I factored out the Mixin. `Event.completed` is actually the only place where we have `.completed`. If anything it would be `Event` that should have `.active`.